CHANGELOG.md

```diff
@@ -28,6 +28,11 @@ _None_
 * CoreData: ensure fetched properties use the right class name.  
   [David Jennes](https://github.com/djbe)
   [#960](https://github.com/SwiftGen/SwiftGen/pull/960)
+* All parsers: fixed unstable output for keys that differed just by case.  
+  [grigorye](https://github.com/grigorye)
+  [David Jennes](https://github.com/djbe)
+  [#891](https://github.com/SwiftGen/SwiftGen/issues/891)
+  [#890](https://github.com/SwiftGen/SwiftGen/pull/954)
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/SwiftGen/SwiftGen/blob/cf4273eb514050f92df6d712f15f18fc07be695c/CHANGELOG.md#L35"></span>Link text is #890 but links points to SwiftGen/SwiftGen#954.</td>
    </tr>
  </tbody>
</table>